docs/site/src/components/AppRouter.js

```diff
@@ -46,7 +46,11 @@ export default function AppRouter() {
       history={hashHistory}
       render={applyRouterMiddleware(useScroll())}
     >
-      <Route title="Material UI" path="/" component={AppFrame}>
+      <Route
+        title="Material UI"
+        path="/"
+        component={(props) => <AppFrame demos={demos} apiDocs={apiDocs} {...props} />}
```

Comment:
They're only passed from AppRouter to AppFrame, which seems like pretty standard usage for props. Personally I would avoid context unless warping data to a grandchild or subordinate component, or to multiple components in the hierarchy. 

Happy to take defer to your guidance here if you disagree strongly, or my reasoning is faulty.